lib/qbo_api/connection.rb

```diff
@@ -1,6 +1,8 @@
 require 'faraday'
-require 'faraday_middleware'
+require 'faraday/response/json'
+require 'qbo_api/raise_http_exception'
 require 'faraday/detailed_logger'
+require 'faraday/multipart'
```

Comment:
Oh,interesting

Reply:
With FaradayMiddleware deprecated, a lot of it was pulled out into gems.

Reply:
ah, faraday 1.x had faraday-middleware as a dependency